src/encoded/static/components/regulome_search.js

```diff
@@ -846,14 +846,16 @@ class RegulomeSearch extends React.Component {
                                 >
                                     <h4>ChIP data</h4>
                                     {(thumbnail === null) ?
-                                        <div>
+                                        <React.Fragment>
                                             <div className="line"><i className="icon icon-chevron-circle-right" />Click to see detailed ChIP-seq data.
-                                                <div>(<b>{chipData.length}</b> result{chipData.length !== 1 ? 's' : ''})</div>
+                                                <React.Fragment>
```

Comment:
Does this make the “result” line move inline with “Click to see detailed ChIP-seq data”? Without the `<div>`, that’s what this looks like, unless `<React.Fragment>` inserts something automatically in this case.

Reply:
Yes, it does move it inline. I needed this <div>! (And the others like it.)